docs/tutorials/fundamentals/viewer.md

```diff
@@ -382,109 +360,126 @@ from scipy import ndimage as ndi
 
 import napari
 
+
 blobs = data.binary_blobs(length=128, volume_fraction=0.1, n_dim=3)
 viewer, image_layer = napari.imshow(blobs.astype(float), name='blobs')
 labeled = ndi.label(blobs)[0]
 viewer.add_labels(labeled, name='blob ID')
 ```
 
-... then clicking on the 2D / 3D button
+then, by clicking the 2D/3D button, you can rotate the image (the camera view of the image) with the mouse to see what it looks like from the side, back, or a different angle. To do this, click on the image and drag the cursor to a new position. This gives something like the following view: 
 
 ![image: 3D_button](../assets/tutorials/3D_button.png)
 
-and rotating the camera view with the mouse gives something like the following
-view:
+and rotating the camera view with the mouse gives something like the following view:
 
 ```{code-cell} python
-:tags: [hide-input]
+:tags: [hide-input]  
 
-# programmatically adjust the camera angle
+# programmatically adjust the camera angle  
 viewer.dims.ndisplay = 3
 viewer.camera.zoom = 2
 viewer.camera.angles = (3, 38, 53)
 nbscreenshot(viewer, alt_text="A rotated 3D view")
 ```
+<!-- This is commented out. ![image: Rotated image](../assets/tutorials/rotated-image.png) -->  
```

Comment:
Why is this commented out? No image is appearing in the built docs. Perhaps we need a dedicated cell, like:
```
```{code-cell} python
:tags: [hide-input]
 nbscreenshot(viewer, alt_text="A 3D view of the blobs sample image.")
\```
```

Reply:
I did  not understand the way things are displayed in VS Code vs html.  I think I have this straightened out.